spec.html

```diff
@@ -35718,22 +35718,20 @@ <h1>
           </dl>
           <emu-alg>
             1. Assert: _p_ is a canonical, unaliased Unicode property name listed in the &ldquo;Canonical property name&rdquo; column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>.
-            1. Assert: _v_ is a property value or property value alias for Unicode property _p_ listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref> or <emu-xref href="#table-unicode-script-values"></emu-xref>.
+            1. Assert: _v_ is a property value or property value alias for the Unicode property _p_ listed in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a>.
             1. Let _value_ be the canonical property value of _v_ as given in the &ldquo;Canonical property value&rdquo; column of the corresponding row.
             1. Return the List of Unicode code points _value_.
           </emu-alg>
-          <p>Implementations must support the Unicode property value names and aliases listed in <emu-xref href="#table-unicode-general-category-values"></emu-xref> and <emu-xref href="#table-unicode-script-values"></emu-xref>. To ensure interoperability, implementations must not support any other property value names or aliases.</p>
+          <p>Implementations must support the Unicode property values and property value aliases listed in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a> for the properties listed in <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>. To ensure interoperability, implementations must not support any other property values or property value aliases.</p>
```

Comment:
You are pointing to the "lastest" Unicode Character Database, which makes this spec "evergreen". However, as a "must support", should you provide a little leniency for recent versions, or the versions as of the release of the ES implementation?

Reply:
We already normatively refer to the latest version of the Unicode standard (for things like whitespace, ID_Start, etc). From the spec,

> Additionally, ECMAScript 2017 mandated always using the latest version of the Unicode standard.

Reply:
For context, this was the relevant PR: https://github.com/tc39/ecma262/pull/620